src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java

```diff
@@ -138,6 +138,22 @@ public enum PersistenceType {
         DB                  //Store only DB
     }
 
+    public enum RecommendationSection {
+        CURRENT_CONFIG(KruizeConstants.JSONKeys.CURRENT),
+        RECOMMENDATION_CONFIG(KruizeConstants.JSONKeys.CONFIG),
+        VARIATION(KruizeConstants.JSONKeys.VARIATION);
+
+        private String name;
+
+        private RecommendationSection(String name) {
+            this.name = name;
+        }
+
+        public String getName() {
+            return this.name;
+        }
+    }
+
```

Comment:
I'd suggest move all of the recommendations constants to the `recommendations` dir. This is bloating the `AnalyzerConstants` even though none of this is used beyond the `recommendations` package

Reply:
Thanks @dinogun for the review, Will make the changes and push the commit to PR.